docs/user/guides/authors.rst

```diff
@@ -16,6 +16,9 @@ and :doc:`/intro/getting-started-with-mkdocs`.
    cross-referencing-with-sphinx
    Link to external projects (Intersphinx) <intersphinx>
    jupyter
+   /guides/technical-docs-seo-guide
    Migrate from rST to MyST <migrate-rest-myst>
    enable-offline-formats
    Using search analytics <search-analytics>
+   /automatic-redirects
+   /science
```

Comment:
hahaha, this page will keep moving around because it's such a hack.

BUT it's noted for later :+1:

Reply:
Yea, I mostly just hid it away. I think it could live under a top-level Tutorials page like `Specific use cases` or something?